src/pages/order-item/order-item.js

```diff
@@ -104,6 +108,17 @@ const OrderItem = ({ id }) => {
       >
         {SAVE_ORDER}
       </Button>}
+      <div className={classes.controlsBlock}>
+        <Link to={config.routes.pathToOrders}>
+          <StandardButton
+              id='back'
+              title='Назад'
+              variant='outlined'
+              onClickHandler={() => {}}
+              data-cy='back-btn'
+          />
+        </Link>
+      </div>
```

Comment:
Please move title to config, and set link to onclickHandler

Reply:
done